### ISSUE TITLE
Slight discrepancy in accuracy numbers

### ISSUE THREAD

**Issue by rtaori** [facebookresearch/FixRes#4]
Hi,

Thanks for this great work! I have taken the models from here and evaluated them using the [standard imagenet pytorch script](https://github.com/pytorch/examples/blob/master/imagenet/main.py); however, the top1 accuracies returned are about 0.35% lower than the number reported in your paper.

On further inspection, it seems like the [resizing logic](https://github.com/facebookresearch/FixRes/blob/master/imnet_evaluate/transforms.py#L28) is slightly different; while the PyTorch default scales the shorter side of an image to the specified size, it seems like the custom logic here always scales the height of the image to the specified size.

Is this intentional? If so, what was the motivation for this change? I read through the paper but couldn't seem to find an answer for this; any clarification you could give would be greatly appreciated. 

Thanks

**Comment by TouvronHugo** [comment]
Hi @rtaori ,

Thank you for your remark,

The Resize used is not an important point of the code, by default we used the one used in https://github.com/facebookresearch/multigrain/ . 
It's normal to get better performance with this one because it is the one used with our fine-tuning method but if you want use another Resize you only have to do the fine-tuning with the Resize that you want to use. 
If you use the code provided in our Github repo you must get the performance reported in the article. 
I implemented the evaluation on Colab (https://colab.research.google.com/drive/1newqzoZlPwY6JlojtzIZIlzm3ezXg9yU) this will probably help you to evaluate the models more easily.

Hugo
